src/components/combobox-item/combobox-item.tsx

```diff
@@ -145,30 +145,39 @@ export class ComboboxItem implements ConditionalSlotComponent, InteractiveCompon
   //
   // --------------------------------------------------------------------------
 
-  renderIcon(isSingle: boolean): VNode {
-    const { icon, disabled, selected, iconFlipRtl } = this;
-    const level = `${CSS.icon}--indent`;
-    const defaultIcon = isSingle ? "dot" : "check";
-    const iconPath = disabled ? "circle-disallowed" : defaultIcon;
-    const showDot = isSingle && !icon && !disabled;
+  renderIcon(iconPath: string): VNode {
+    return this.icon ? (
+      <calcite-icon
+        class={{
+          [CSS.custom]: !!this.icon,
+          [CSS.iconActive]: this.icon && this.selected,
+          [`${CSS.icon}--indent`]: true
```

Comment:
Please move this class to [`resources.ts`](https://github.com/Esri/calcite-components/blob/933637102b40994e70c33d0189fe438f574629fe/src/components/combobox-item/resources.ts#L3) instead of dynamically creating it here.